src/common.cc

```diff
@@ -120,29 +120,31 @@ WabtResult wabt_read_file(const char* filename,
                           size_t* out_size) {
   FILE* infile = fopen(filename, "rb");
   if (!infile) {
-    fprintf(stderr, "unable to read file: %s\n", filename);
+    char msg[128];
```

Comment:
I'm OK with this for now, since we'll likely replace a lot of the snprintf code since we're switching to C++. Do you mind adding a TODO for supporting messages longer than 128, though?